index.html

```diff
@@ -4600,14 +4649,16 @@ <h2 id="AudioBuffer">
           <dd>
             <p>
               The sample-rate for the PCM audio data in samples per second.
+              This MUST return the value of <var>[[sample rate]]</var>.
```

Comment:
Yes. It's mostly just a bunch of repetitive manual work, and it's non blocking for CR, it seems.